docs/topic/troubleshooting.md

```diff
@@ -34,18 +34,15 @@ This should revert the Helm deployment to the previous revision (the one just be
 
 Prometheus collects Kubernetes metrics for our grafana charts and often requires a lot of memory during startup.
 If the server runs out of memory during the startup process, this presents itself as a "Timed out waiting for condition" error from `helm` on the command line, and `OOMKilled` events in the prometheus server pod events before the pod reverts to `CrashLoopBackOff` status.
```

Comment:
```suggestion
If the server runs out of memory during the startup process, this presents itself as a "Timed out waiting for condition" error from `helm` on the command line, and `OOMKilled` events in the prometheus server pod events before the pod reverts to `CrashLoopBackOff` status.
This might also block our CI/CD system from updating the hubs on the cluster if the support chart requires an upgrade after a Pull Request is merged.
```